content/docs/install/completion.md

```diff
@@ -100,39 +96,36 @@ install and configure shell completion.
    fi
    ```
 
-4. Finally, `source ~/.bashrc` or open a new terminal to activate it.
+4. Finally, `source ~/.bashrc` or open a new terminal to activate completions.
 
 ## Zsh completion
 
-The DVC specific completion script is located in this path of our main
-repository:
-[dvc/scripts/completion/dvc.zsh](https://github.com/iterative/dvc/blob/master/scripts/completion/dvc.zsh)
+Place the completion script in a directory included in `$fpath`. Zsh expects the
+file name to be `_dvc`.
 
-Place the completion script in a directory included in `$fpath`, the file name
-should be `_dvc`.
+A nice way to ensure completions are always up-to-date is to refresh them
+whenever a new terminal is opened.
 
-For example:
+1. Edit `~/.zshrc` and add the following lines to the top of the file:
 
-```dvc
-$ sudo wget \
-    -O /usr/local/share/zsh/site-functions/_dvc \
-    https://raw.githubusercontent.com/iterative/dvc/master/scripts/completion/dvc.zsh
-```
+   ```dvc
+   mkdir -p ~/.local/completions
+   fpath=($fpath ~/.local/completions)
+   dvc completion -s zsh > ~/.local/completions/_dvc
+   ```
```

Comment:
This might turn ugly fast, because not everyone is installing dvc into the global env. I would rather not recommend this.

Reply:
Or at least would put a disclaimer.

Reply:
Hmm there's an annoying trade-off between staying up-to-date versus explaining how to work with various virtualenvs in detail (e.g. using `CONDA_PREFIX/etc/conda/activate.d/env_vars.sh`).

I think the solution may be to install once hard-coded (as with the original instructions) and then in addition have a section for eager installation recommending things like `echo 'which dvc && eval "$(dvc completion -s bash)"' >> ~/.bash_completion`

Reply:
oh what about `dvc completion -s bash --eager` printing `which {sys.executable} && eval "$({sys.executable} -m dvc completion -s bash)"`?

Then we could just recommend:

```
$ dvc completion -s bash --eager >> ~/.bash_completion
```

Reply:
@casperdcl Let's not overcomplicate it :slightly_smiling_face:

Reply:
just updated with two slightly different ideas (one for bash and one for zsh) - let me know if you like either (or neither).